.travis.yml

```diff
@@ -1,9 +1,20 @@
 sudo: false
 language: python
+
 python:
   - 3.5
+
 services:
-    - postgresql
+  - postgresql
+  - docker
+
+env:
+  global:
+    - secure: "" # DOCKER_EMAIL
```

Comment:
How does that _secure_ works? Should I set these variables in Travis CI? Drop a line at telegram.me/cuducos so we can sort that out.